src/Integration/Connection/UI/OrganizationSelectionWindow.xaml

```diff
@@ -41,46 +42,98 @@
     </vsui:DialogWindow.Resources>
 
     <!--  Design  -->
-    <StackPanel Style="{StaticResource DialogContent}">
-        <TextBlock Margin="0,0,0,8"
-                   FontStyle="Italic" Text="Please select the organization you want to bind to." TextWrapping="Wrap" />
-
-        <StackPanel Orientation="Horizontal">
-            <TextBlock VerticalAlignment="Center" Text="Organizations" />
-            <ComboBox x:Name="OrganizationComboBox"
-                      Width="250" Margin="12,0,0,0"
-                      HorizontalContentAlignment="Stretch"
-                      IsEditable="True" IsTextSearchCaseSensitive="False" IsTextSearchEnabled="True"
-                      TextSearch.TextPath="Name">
-                <ComboBox.ItemsPanel>
-                    <ItemsPanelTemplate>
-                        <VirtualizingStackPanel />
-                    </ItemsPanelTemplate>
-                </ComboBox.ItemsPanel>
-                <ComboBox.ItemContainerStyle>
-                    <Style TargetType="{x:Type ComboBoxItem}">
-                        <Setter Property="Width" Value="230" />
-                    </Style>
-                </ComboBox.ItemContainerStyle>
-                <ComboBox.ItemTemplate>
-                    <DataTemplate DataType="{x:Type models:SonarQubeOrganization}">
-                        <TextBlock TextTrimming="CharacterEllipsis">
-                            <Run Text="{Binding Name, Mode=OneWay}" />
-                            <Run Text="(" /><Run Text="{Binding Key, Mode=OneWay}" /><Run Text=")" />
-                        </TextBlock>
-                    </DataTemplate>
-                </ComboBox.ItemTemplate>
-            </ComboBox>
-        </StackPanel>
-
-        <Separator Style="{StaticResource DialogSeparator}" />
-
-        <StackPanel VerticalAlignment="Center" Style="{StaticResource DialogButtonsPanel}">
-            <vsui:DialogButton Click="OnOkButtonClick" Content="OK" IsDefault="True"
-                               IsEnabled="{Binding ElementName=OrganizationComboBox,
-                                                   Path=SelectedItem,
-                                                   Converter={StaticResource NotNullConverter}}" />
-            <vsui:DialogButton Content="Cancel" IsCancel="True" />
-        </StackPanel>
-    </StackPanel>
+    <!-- Note: we're using grids inside the TabItem with IsSharedSizeScope to make sure the tabs are the same size
+         (otherwise they will be different and the dialogue box will resize when the user switches tabs) -->
+    <TabControl Grid.IsSharedSizeScope="True" >
+        <TabItem Header="Your organizations">
```

Comment:
Previously, the UI consisted of a combobox bound to the list of organizations fetched from the server.
The changes to the webclient submodule (above) limited the list of organizations to just the orgs the user belongs to, rather than all orgs with public projects.
However, that meant there was no way using the old dialogue for the user to select an organization they don't belong to. 

To fix this, I've added another tab to the UI to allow the user to type the organization key into a textbox. The UI is isn't very polished but it's good enough for this bug fix, and we don't want to spend more time on this at present.